app/helpers/validation_helper.rb

```diff
@@ -88,6 +88,7 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     'Fully Vaccinated',
     'Deceased',
     'Duplicate',
+    'Refused Monitoring',
     'Other'
   ].freeze
 
```

Comment:
Just a question about the history items generated (I know you didn't change any of this @timwongj)...

It seems too much to have both these messages, especially with the one on the top being contradictory
<img width="1696" alt="Screen Shot 2022-01-04 at 5 57 18 PM" src="https://user-images.githubusercontent.com/35042815/148135290-cc102686-63ba-4a84-8f15-47329fc1861b.png">

Would just the second one suffice?

Reply:
I agree. Can you create a ticket on the backlog?